packages/plugins/experimental/plugin-automation/src/components/TriggerEditor/TriggerEditor.tsx

```diff
@@ -26,6 +26,29 @@ export type TriggerEditorProps = {
   onCancel?: () => void;
 };
 
+const PayloadInput = (props: InputProps & { property: string }) => {
+  const { t } = useTranslation(AUTOMATION_PLUGIN);
+  const inputProps = useInputProps(['payload', props.property]);
+  return (
+    <div role='none' className='flex items-center mt-2 gap-1'>
+      <div role='none' className='flex-1'>
```

Comment:
do you need a wrapping div here?

Reply:
I just extracted this from below. It's a mess and the Form should really be handling this type of input directly, which is what the todo below is about.